aws/ci_build.rake

```diff
@@ -223,8 +219,8 @@ def upgrade_frontend(name, host)
     HipChat.log "<b>#{name}</b> (#{host}) failed to upgrade, removing from rotation.", color: 'red'
```

Comment:
yes, the `bin/deploy_frontend/register_with_elb.sh` script will be called at the end of a successful upgrade- see line [201](https://github.com/code-dot-org/code-dot-org/pull/8312/files#diff-3b3681eea9267282c5abb86750b4b105R201) where it's added to the ssh command generated.